luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm

```diff
@@ -47,7 +47,7 @@ <h3>Freifunk-Assistent</h3>
 <div class="clear"></div>
 </div>
 <div class="cbi-page-actions">
-  <a class="cbi-button cbi-button-apply" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/generalInfo")%>">Back</a>
-  <a class="cbi-button cbi-button-apply" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/cancel")%>">Cancel</a>
+  <a class="cbi-button cbi-button-apply" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/generalInfo")%>"><%= translate("Back") %></a>
+  <a class="cbi-button cbi-button-apply" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/cancel")%>"><%= translate("Cancel") %></a>
```

Comment:
You may use the shortcut `<%:Text to translate%>` like shown [here](https://github.com/openwrt/luci/wiki/Templates#translating).
But this is not too important. This way is perfectly valid too.

Reply:
In this case, I kept the style used throughout the source file.

```
~/src/falter/falter-packages$ git grep "<%= trans" 
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:<%= translate("Als naechstes kommen die Einstellungen für das Freifunk-Netzwerk. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:        <%= translate("Am Freifunk-Netz teilnehmen") %>
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:        <%= translate("Erweitere das Freifunk-Netz. Du benoetigst dazu eine \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:        <%= translate("Am Freifunk-Netz teilnehmen und Internet teilen") %>
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:        <%= translate("Unterstütze die Community, indem du dein Internet im Freifunk-Netzwerk freigibst. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:  <a class="cbi-button cbi-button-apply" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/generalInfo")%>"><%= translate("Back") %></a>
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/decide.htm:  <a class="cbi-button cbi-button-apply" href="<%=luci.dispatcher.build_url("admin/freifunk/assistent/cancel")%>"><%= translate("Cancel") %></a>
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/reboot.htm:      <%= translate("Du hast alle Einstellungen geschafft. Dein Router wird jetzt neu gestartet. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/reboot.htm:      <%= translate("Bitte beachte, dass der Router dann unter seiner neuen Freifunk-IP zu \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/reboot.htm:      <%= translate("Es gibt auch Services im Freifunk-Netz. Ein Liste kannst du dir \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/enableStats.htm:    <%= translate("Wenn Du hier den Haken setzt, werden statistische Daten (RAM-Auslastung, CPU-Auslastung, Bandbreitennutzung etc.) dieses Routers erhoben und in der Oberflaeche des Routers angezeigt sowie an monitor.berlin.freifunk.net geschickt. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/generalInfo.htm:    <%= translate("Wenn du möchtest, kannst du hier deine Kontaktdaten eingeben. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/ipinfo.htm:    <%= translate("Die SSID des Freifunk-WLAN-APs solltest du auf <em>berlin.freifunk.net</em> lassen, \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/ipinfo.htm:    <%= translate("Die DHCP-Einstellung regelt die Vergabe von IP-Adressen an Clients deines \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/meshipinfo.htm:    <%= translate("Im Folgenden werden die IPs eingetragen, die du vom Wizard unter \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/shareBandwidth.htm:    <%= translate("Die Knoten des Freifunk-Netzes leiten die Daten auf der Basis des kuerzesten Wegs und der hoechsten Bandbreite weiter. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/vpncertreupload.htm:    <%= translate("Schluessel loeschen und erneut hochladen") %>
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/vpninfo.htm:    <%= translate("Im Folgenden wird dein Schluessel zum Tunnel/VPN gebraucht. \
luci/luci-app-ffwizard-falter/luasrc/view/freifunk/assistent/snippets/welcome.htm:    <%= translate("Herzlichen Glueckwunsch! \
```